src/analysis/normed_space/multilinear.lean

```diff
@@ -288,7 +288,7 @@ instance has_op_norm : has_norm (continuous_multilinear_map 𝕜 E G) := ⟨op_n
 
 lemma norm_def : ∥f∥ = Inf {c | 0 ≤ (c : ℝ) ∧ ∀ m, ∥f m∥ ≤ c * ∏ i, ∥m i∥} := rfl
 
--- So that invocations of `real.Inf_le` make sense: we show that the set of
+-- So that invocations of `cInf_le` make sense: we show that the set of
```

Comment:
```suggestion
-- So that invocations of `le_cInf` make sense: we show that the set of
```
From the sequel, it seems that this is used with `le_cInf`.